### PR TITLE
Remove references to Classic Editor

### PR DESCRIPTION
**Pull request by plttn**

### CODE REVIEW

**Review by lxjv — CHANGES_REQUESTED** [review]
can you remove any mentions to the editor being advanced as there is only one editor now? (ill go around and fix links after)

File: info/Profile & Web/classic-editor.md [hunk 1]
```diff
@@ -5,4 +5,4 @@ Last Updated: 2023-01-14 4:09 PM EST
 
 # Classic Web Editor
 
-We’re retiring our Classic Web Editor on January 31, 2023. [Read more here.](https://omglol.news/2023/01/14/so-long-classic-web-editor)
\ No newline at end of file
+We’re retiring our Classic Web Editor on January 31, 2023. [Read more here](https://omglol.news/2023/01/14/so-long-classic-web-editor).
```
Comment (Vukkyy) [inline info/Profile & Web/classic-editor.md:8]: It's already retired, so maybe this page should be removed/redirected?

(disclaimer: i am not a staff.lol)

**Comment by byemc** [comment]
I vote for redirecting “/classic-editor” to “/editor” and renaming “/advanced-editor.md” to just “/editor.md"

~bye

**Review by byemc — APPROVED** [review]

**Review by lxjv — APPROVED** [review]
nice :D

**Comment by lxjv** [comment]
> I vote for redirecting “/classic-editor” to “/editor” and renaming “/advanced-editor.md” to just “/editor.md" - @byemc 

me too, ill send a PR in

**Pull request merged**